ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/build/BuildSourceViewerConfiguration.java

```diff
@@ -38,6 +42,9 @@ public class BuildSourceViewerConfiguration extends ChangeAwareSourceViewerConfi
 	private BasePDEScanner fCommentScanner;
 	private BasePDEScanner fPropertyValueScanner;
 
+	private ContentAssistant fContentAssistant;
+	private BuildPropertiesContentAssistProcessor fContentAssistantProcessor;
```

Comment:
`BuildPropertiesContentAssistProcessor fContentAssistantProcessor` can be a local variable in `getContentAssistant()`

Reply:
sure, done.